components/shared/IndexIssueListItem.js

```diff
@@ -19,33 +20,42 @@ function parseBody(body) {
   return [description, img];
 }
 
-export default function IndexIsssueListItem({ type, issue }) {
+export default function IndexIssueListItem({ issue }) {
   const [description, img] = parseBody(issue.body);
+
+  const type = issue.labels.includes("Product Index") ? "product" : "project";
+
```

Comment:
Moved this here so we don't have to have this code over and over again in `pages/index.js`